### PR TITLE
Release 13.3.0

### PR DESCRIPTION
**Pull request by wyardley**
Includes

- #978 
- #975 
- #977 
- #974 
- #969 
- #980
- #915 
- #981 
- #982

### CODE REVIEW

**Review by wyardley — COMMENTED** [review]
Note: I got some slight differences in other releases with my local version of GCG. I just left those sections alone and included the 13.3 section, but if someone prefers, I can push the file as-generated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
...
+- update puppet-systemd upper bound to 8.0.0 [\#977](https://github.com/voxpupuli/puppet-rabbitmq/pull/977) ([TheMeier](https://github.com/TheMeier))
+- Harden codebase and add documentation stubs [\#974](https://github.com/voxpupuli/puppet-rabbitmq/pull/974) ([zilchms](https://github.com/zilchms))
+
+**Fixed bugs:**
+
+- add a workaround for rabbitmq\_vhost when running with --noop or --tags [\#969](https://github.com/voxpupuli/puppet-rabbitmq/pull/969) ([bugfood](https://github.com/bugfood))
+
+**Closed issues:**
+
+- Literal Type errors [\#973](https://github.com/voxpupuli/puppet-rabbitmq/issues/973)
```
Comment (bastelfreak) [inline CHANGELOG.md:24]: added the `skip-changelog` label, please regenerate

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
...
+- Failing with "Language validation logged 2 errors" [\#971](https://github.com/voxpupuli/puppet-rabbitmq/issues/971)
```
Comment (bastelfreak) [inline CHANGELOG.md:25]: added skip-changelog here as well

**Comment by wyardley** [comment]
FWIW, it seems like sometimes GCG is now including both an issue and the PR that fixed / closed it if both are tagged as 'enhancement'. I'm using a combination of removing labels or adding `skip-changelog` to fix that.

The other issue I'm seeing is some older entries where GH usernames changed or are no longer extant. I'm letting this update the one where the user changed, but leaving the one that changed to "ghost" as it was for now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -643,7 +663,7 @@ Debian 8 is EOL since some months now. Release 10.2.0 will be the last one with
 - \[Bugfix\] convert $ssl\_depth from string to integer [\#539](https://github.com/voxpupuli/puppet-rabbitmq/pull/539) ([bastelfreak](https://github.com/bastelfreak))
 - Extend rabbitmqadmin config template with SSL options. [\#526](https://github.com/voxpupuli/puppet-rabbitmq/pull/526) ([justahero](https://github.com/justahero))
 - \[MODULES-4223\] don't set NODE\_PORT and NODE\_IP\_ADDRESS if ssl\_only [\#524](https://github.com/voxpupuli/puppet-rabbitmq/pull/524) ([JAORMX](https://github.com/JAORMX))
-- \[MODULES-3733\] rabbitmq provider env\_path does not locate ruby gem installed puppet binary [\#517](https://github.com/voxpupuli/puppet-rabbitmq/pull/517) ([lowstorage](https://github.com/lowstorage))
+- \[MODULES-3733\] rabbitmq provider env\_path does not locate ruby gem installed puppet binary [\#517](https://github.com/voxpupuli/puppet-rabbitmq/pull/517) ([growthstock](https://github.com/growthstock))
```
Comment (wyardley) [inline CHANGELOG.md:666]: note: this is just a GH username change; assume we should go ahead and keep this in so future runs don't include this line again?

Comment (bastelfreak) [reply CHANGELOG.md:665]: yes we usually keep this

**Comment by bastelfreak** [comment]
@wyardley good and bad news! good: https://github.com/voxpupuli/puppet-rabbitmq/pull/915 passes now. bad: we're not compatible for puppet 8 yet, so I reverted 701ab63f6834e994e0829f0c579628d17052aa29. I wanted to do that in the modulesync branch but did it on master. And I'm not sure how we should properly address this. I can revert the revert and provide that as a new PR?

**Comment by wyardley** [comment]
Wow, thanks for getting modulesync working. Do you want to squash the other commits in 915, and add the revert as another commit there? Or a separate revert commit is also fine

**Comment by bastelfreak** [comment]
lets see, I will try to add official puppet 8 support

**Pull request merged**